axelrod/strategies/dbs.py

```diff
@@ -0,0 +1,354 @@
+from axelrod.actions import Actions, Action
+from axelrod.player import Player
+
+
+C, D = Actions.C, Actions.D
+
+def action_to_int(action):
+    return (1 if action==C else 0)
+
+
+class DBS(Player):
+    """
+    Desired Belief Strategy as described in:
+    Accident or Intention: That Is the Question (in the Noisy Iterated Prisoner's Dilemma) by Tsz-Chiu Au and Dana Nau from University of Maryland
+    http://www.cs.utexas.edu/%7Echiu/papers/Au06NoisyIPD.pdf
+
+    A strategy that learns the opponent's strategy, and use symbolic noise de-
+    tection for detecting whether anomalies in player’s behavior are deliberate or accidental, hence performing quite well in noisy tournaments.  
+
+    From the learned opponent's strategy, a tree search is used to choose the best move
+
+
+
+
+    Default values for the parameters are the suggested values in the article 
+    When more noise you can try to diminish violation_threshold and rejection_threshold
+
+    Parameters:
```

Comment:
Please see our formatting for other strategies for the parameter docs (and thanks for including them)